scripts/cluster/join.py

```diff
@@ -338,34 +338,38 @@ def reset_current_dqlite_installation():
     if len(my_ep) > 0 and "127.0.0.1" not in my_ep[0]:
         for ep in other_ep:
             try:
-                subprocess.check_output("curl -X 'DELETE' https://{}/cluster/{} --cacert {} --key {} --cert {}  -k -s"
+                subprocess.check_output("curl -X DELETE https://{}/cluster/{} --cacert {} --key {} --cert {}  -k -s"
```

Comment:
Out of interest, why not use `requests`?

Reply:
This comment is addressed in https://github.com/ubuntu/microk8s/pull/1220

Yes, requests looks better. We switched to that.